### ISSUE TITLE
accuracy

### ISSUE THREAD

**Issue by ahworld22** [ZelongZeng/PLCD#6]
Hello，i would like to ask why when I go to the training step of 2 and 3 ,the accuracy always be very very low at 1%.But when I train it at step 1,the accuracy is very high like 90%

**Comment by ZelongZeng** [comment]
Hi, could you share the results of your experiments and datasets?

**Comment by ahworld22** [comment]
> Hi, could you share the results of your experiments and datasets?

Yes, I used the model with the highest accuracy from the previous step as the teacher for training. I trained the second step on the model that achieved an accuracy of 90% in the first step, but the final accuracy was very low. I don't know what caused this. However, I noticed that I didn't do the prepare_limited_view step, could it be the problem?
[plcd训练2.txt](https://github.com/user-attachments/files/17306102/plcd.2.txt)

**Comment by ZelongZeng** [comment]
The ground-to-drone task is quite challenging, and the test results are unlikely to reach 90%. Could you tell me which dataset you used for testing and what the 90% metric refers to? Is it Top-1 accuracy or mAP?

**Comment by ahworld22** [comment]
> The ground-to-drone task is quite challenging, and the test results are unlikely to reach 90%. Could you tell me which dataset you used for testing and what the 90% metric refers to? Is it Top-1 accuracy or mAP?

I use the dataset of university,and the accuracy is about 80% not 90%(sorry I misstate it):
![image](https://github.com/user-attachments/assets/27cf5fda-096e-4357-912b-c6610b0cc4d7)

**Comment by ahworld22** [comment]
> The ground-to-drone task is quite challenging, and the test results are unlikely to reach 90%. Could you tell me which dataset you used for testing and what the 90% metric refers to? Is it Top-1 accuracy or mAP?

hello，I'm looking forward to your reply!I send you the trainning result as follow.I use the dataset of university,and the accuracy is about 80% not 90%(sorry I misstate it).And I use the best epoch of step one to train step two, it only reaches 1% acurracy,what's the problem?However, I noticed that I didn't do the prepare_limited_view step, could it be the problem?
![image](https://github.com/user-attachments/assets/5ca1b5d6-6ea9-4b67-85e5-81242c2f6fe8)

**Comment by ZelongZeng** [comment]
Hi, this is the result on the training-set, you need to test your model on the test-set.

**Comment by ahworld22** [comment]
> Hi, this is the result on the training-set, you need to test your model on the test-set.

I know but it should be the model of step3 that be used to test right?I see the trainning result of step 2 be very low I want to know what is the problem of this!

**Comment by ZelongZeng** [comment]
Normally, this situation shouldn't occur. You can check whether the loss curve is following a normal downward trend.

**Comment by ahworld22** [comment]
In step1 it should not use the pre-train model you given right？Also I would like to know whether the test and evaluate part will output the visual result or it only output the data analysis.



---- Replied Message ----
| From | Zelong ***@***.***> |
| Date | 10/13/2024 21:51 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [ZelongZeng/PLCD] accuracy (Issue #6) |

Normally, this situation shouldn't occur. You can check whether the loss curve is following a normal downward trend.

**Comment by ahworld22** [comment]
And I saw the loss is continuously high！I don't know whats the problem of that.



---- Replied Message ----
| From | Zelong ***@***.***> |
| Date | 10/13/2024 21:51 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [ZelongZeng/PLCD] accuracy (Issue #6) |

Normally, this situation shouldn't occur. You can check whether the loss curve is following a normal downward trend.

**Comment by ZelongZeng** [comment]
If the values are unusually high or not decreasing, there might be an issue. Many reasons could cause this situation. You can check if there are any abnormalities in the generated gradients. Sometimes, differences in environment versions can lead to such problems.
